src/stylesheets/tk-form-fields.scss

```diff
@@ -1,16 +1,25 @@
 @charset "utf-8";
 @import "variables";
-@import "utilities";
-
+// External dependencies
 @import '@sharedweb/tk-core-styles/dist/index';
 @import '@sharedweb/tk-typography/dist/index';
 @import 'bootstrap-sass/assets/stylesheets/bootstrap/forms';
 @import 'bootstrap-sass/assets/stylesheets/bootstrap/buttons';
-
-/* Form fields partials */
+// Form fields partials
```

Comment:
Or...it should be documented that when importing this module, wrap the import in a non-global namespace, such as:

```scss
.my-module {
  @import 'tk-form-fields/src/stylesheets/tk-form-fields';
  @include tkff;
}
```